Quickhooks/README.md

```diff
@@ -8,4 +8,4 @@ Quickhooks are fast-printing, two part hook/hangar solutions designed for quick
 
 ## Printing notes
 
-Quickhooks should be suitable for fast printing, set your slicer up for maximum perimeters to ensure a strong printed result.  The retainer is press fit into the quickhook body.  I use a clamp to apply a steady force to press the retainer into place.
+Quickhooks should be suitable for fast printing, set your slicer up for maximum perimeters to ensure a strong printed result.  The retainer is press fit into the quickhook body. I use a clamp to apply a steady force to press the retainer into place. A rubber mallet is another good way to press fit together a quickhook and a retainer.

```

Comment:
I've also found a $6 jeweler's hammer from Harbor Freight works great for final assembly. I'll use vice grip pliers initially to try and set a pin, then just set on a flat surface to hammer down using the nylon side. Also works well to lock in pieces on the board itself if it's a really tight fit.

https://www.harborfreight.com/hand-tools/hammers-pry-bars/mallets/jewelry/double-sided-mallet-with-wooden-handle-98285.html